src/browser/components/SettingsPage.jsx

```diff
@@ -6,39 +6,31 @@
 /* eslint-disable react/no-set-state */
 
 import React from 'react';
-import {Button, Checkbox, Col, FormGroup, Grid, HelpBlock, Navbar, Radio, Row} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import {Button, Checkbox, Col, FormGroup, Grid, HelpBlock, Navbar, Radio, Row, Modal} from 'react-bootstrap';
 
-import {ipcRenderer, remote} from 'electron';
+import {remote} from 'electron';
 import {debounce} from 'underscore';
 
-import Config from '../../common/config';
-
```

Comment:
Redundant now that this component is controlled through MainPage, which has it's own `Config`.